feature/experimental/tunnel/otg_tests/tunnel_interface_based_resize_test/README.md

```diff
@@ -0,0 +1,37 @@
+
+
+
+ ## TUN-1.6: Tunnel End Point Resize - Interface Based GRE Tunnel
+
+ ## Summary
+
+ *   Validate of interface based GRE tunnel end point reduction and increment test with load balaning.
+
+ ## Procedure
```

Comment:
Why is this not matching the originally shared testplan?

Reply:
Now made it as similar to the test plan